### DISCUSSION TITLE
Pyvista mesh Modification

### DISCUSSION THREAD

**Discussion by AnthonyB08** [dpploy/engy-5310#34] [Q&A]
After importing my .vtu file with pyvista I would like to adjust the mesh boundaries to make the radial axis dimensions. How do I visualize the data as an array so that I may modify it accordingly?

As seen below, the R-axis goes from 0.0 to 7.5 which is the radius of my domain. However, MOOSE solved the problem as an axis symmetrical domain revolving around the X-axis. MOOSE, when exporting the data, does not change the mesh to axis-symmetric as seen below. Therefore, if I can modify the bounds so that each increment, in the radial direction, is divided by the max value. I can represent the axis as a dimensionless quantity of x/D.  

![image](https://user-images.githubusercontent.com/55299169/116398281-9dba7200-a7f5-11eb-99e6-de7367ccc763.png)

**Comment by dealmeidavf** [comment]
There is a way to scale the domain through `pyvista` but I don't remember at the moment. Will post here if I come across it.

**Reply by AnthonyB08** [reply]
Scaling only changes the size of the mesh relative to the window it is displayed on. I need to modify the actual data of the mesh for the bounds to be dimensionless.

**Reply by dealmeidavf** [reply]
I think I have seen an example with scaling the field; they may call a scalar. Haven't come across the example but when I do I will add a note here and use in the course notebooks to save the usage.
